client/cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3000"
+}
```

Comment:
Actuellement les tests Cypress se font sur la DB de dev locale, et donc éventuellement en ajoutant à chaque fois des données. Est-ce que ça sera pertinent de donner un moyen d'utiliser une DB "vide" ? Ajd c'est faisable avec par ex `createdb catalogage-test` puis en passant `DBNAME=catalogage-test`. Je ne sais pas si on devrait le faire systématiquement

Reply:
C'est bien vu, mais après réflexion, c'est soit inutile soit impossible :
- en local, je pense qu'on tourne la plupart du temps des tests sur le serveur lancé en cours (donc c'est trop tard pour y mettre `DBNAME=...`)
- en CI effectivement la DB est vierge à chaque run...

Je te propose de laisser tel quel, et on itère selon nos cas d'usage.